discord/ext/commands/converter.py

```diff
@@ -294,6 +294,8 @@ class ColourConverter(Converter):
             arg = arg[1:]
         try:
             value = int(arg, base=16)
+            if not 0 <= value <= 16777215:
+                raise ValueError()
```

Comment:
Instead of double raising (`ValueError` -> `BadArgument`), raise `BadArgument` here.